scripts/beamline_configuration.py

```diff
@@ -15,8 +15,8 @@
 
 # Set up a RunEngine and use metadata backed by a sqlite file.
 from bluesky import RunEngine
-from bluesky.utils import get_history
-RE = RunEngine(get_history())
+from bluesky.utils import PersistentDict
+RE = RunEngine(PersistentDict(os.path.expanduser('~')))
```

Comment:
Per conversation at group meeting, this needs to be pointed at an _empty_ directory because it will try to interpret all files in the directory as msgpack-encoded files. If the directory does not exist, it will be created. The path

```py
from pathlib import Path
Path('~/.bluesky_history').expanduser()
```

Reply:
I like the suggestion, and implemented it via 7a279cb, wrapping `Path` into `str(...)`.